servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -830,7 +830,7 @@ void loadConfigurationSettings() {
 
         final String skipDomains = System.getProperty(
                 ZMSConsts.ZMS_PROP_VALIDATE_SERVICE_MEMBERS_SKIP_DOMAINS, "");
-        validateServiceMemberSkipDomains = new HashSet<>(Arrays.asList(skipDomains.split(",")));
+        validateServiceMemberSkipDomains = new ArrayList<>(Arrays.asList(skipDomains.split(",")));
```

Comment:
No need to create a new ArrayList<> here. Arrays.asList already does the job and returns an ArrayList. So just call it:

validateServiceMemberSkipDomains = Arrays.asList(skipDomains.split(","));

Reply:
I considered writing it that way, but Arrays.asList() and new ArrayList return different types. ([ref](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-))
As a result, there may be some confusion about this behavior in the future.
However, I am not too concerned about it, so I think Arrays.asList() is fine.
Thanks for pointing this out.